webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -37,16 +37,27 @@ import {
 import {Box} from "@mui/material";
 import {RepoError} from "./error";
 import { getContentType, getFileExtension, FileContents } from "./objectViewer";
+import {OverlayTrigger} from "react-bootstrap";
+import Tooltip from "react-bootstrap/Tooltip";
 
 
 const README_FILE_NAME = 'README.md';
 const REPOSITORY_AGE_BEFORE_GC = 14;
 
-const ImportButton = ({variant = "success", enabled = false, onClick}) => {
+const ImportButton = ({variant = "success", onClick, config }) => {
+    let tip = config.import_support ? "Import data from a remote source" :
```

Comment:
If `tip` isn't set elsewhere, it should be `const`, not `let`

Reply:
Done.